packages/fakta-beregning/src/BeregningFaktaIndex.tsx

```diff
@@ -71,6 +72,17 @@ const skalVurderes = (bg: Beregningsgrunnlag, vilkårsperioder: vilkarperiodeTsT
 
 type AksjonspunktDataDef = SubmitBeregningType[];
 
+function konverterTilNyeAvklaringsbehovKoder(beregningsgrunnlag: Beregningsgrunnlag[]) {
```

Comment:
`const konvertertBeregningsgrunnlag = [...beregningsgrunnlag];` på neste linje, bruke det i `for`-løkkene og returnerer det til slutt gjør vel at du slipper `@ts-ignore`

Reply:
Ignoren er også pga assignet av readonly parameter. Denne metoden her skal jo berre ligge her i ein overgangsfase til vi har gått over til å sende nye koder fra kalkulus. Tenkte derfor at det var greit å endre på input for å slippe å måtte gjere endringer for mange stader.

Reply:
Skjønner. Kjør på